taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/TestCancelledTaskRetrieval.java

```diff
@@ -38,8 +38,8 @@
   @Autowired private JobExecutor jobExecutor;
 
   @WithAccessId(
-      userName = "teamlead_1",
-      groupNames = {"admin"})
+      user = "teamlead_1",
+      groups = {"admin"})
```

Comment:
we should try to avoid admin in the integration tests, because this disabled all authorisation checks. We should a value which could be valid in production as well.